docs/workflows.md

```diff
@@ -101,63 +101,83 @@ Examples:
 
-After creating a Workflow, it can be tested using the `Test Workflow` button that looks like a play button in the right part of the page, on the workflow line that you want to test.
+If the workflow has any error, the error message will be displayed at the bottom of the page.
 
-If the workflow doesn't have input parameters, when trying to test it, a pop-up will appear that warns about workflow not having required parameters to be executed. After this pop-up, hitting the `Run` button will execute the workflow and in the bottom of the page will appear a message displaying how long it took for the workflow to execute and if output parameters are set, they will be displayed there, with the returned values.
+### `Cloning Workflows`
 
-If the workflow does have input parameters, the previous warning pop-up will now contain the input parameter(s) name(s) and for each one an empty field to be completed. The workflow executes and returns the same thing as before after hitting the `Run` button.
+In the event there is an existing workflow that has logic close to what is needed in a new workflow, it can be duplicated and then edited to implement the new logic needed. In the main Workflow list, at the end of the row where the Workflow buttons are located, the last button (when hovered, the tooltip label is **'Duplicate'**) will immediately create a new workflow named with the name of the duplicated workflow appended with **'Copy'** (e.g. 'Send Email to User Copy'). If the same (original) workflow is duplicated again, the name will have **'Copy 2'** appended. The number will increment again as many times as the original workflow is duplicated.
 
-If the workflow has any error, the error message will be displayed in the bottom of the page.
+After duplicating a workflow, it will have exactly the same **Input Parameters**, **Process Flow**, and **Output Parameters** as the original workflow and will function the same. It should be edited to create a new process with the desired outcome and renamed appropriately.
 
 ## `UI Logic`
 
-After you can create the Workflow you have a very powerful tool on your hands and need to integrate it in different scenarios. It can be used as an action called `Execute Workflow` which calls it and execute the whole logic behind it  used on every product that supports the ability to execute an action, such as Form, Listing, Automation, API, Guides.
+After you create the Workflow, you have a very powerful tool on your hands and need to integrate it in different scenarios. It can be used as an action called `Execute [Name of Workflow]` which calls it and executes the logic behind it.  Workflow actions are available on every product that supports the ability to execute an action, such as Forms, Listings, Automation, API, and Guides.
```

Comment:
```suggestion
After you create the Workflow, you have a powerful tool on your hands and can now integrate it in different scenarios. It can be used as an action called `Execute [Name of Workflow]`, which calls it and executes its logic.  Workflow actions are available on every product that supports the ability to execute an action, such as Forms, Listings, Automation, API, and Guides.
```